pkg/webui/console/views/device-add/manual/form/warning-tooltip.js

```diff
@@ -0,0 +1,84 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+import { isNumber } from 'lodash'
+
+import Tooltip from '@ttn-lw/components/tooltip'
+import Icon from '@ttn-lw/components/icon'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+import style from './form.styl'
+
+const isEqual = (current, desired) => {
+  if (isNumber(current) || isNumber(desired)) {
+    return Number(current) === Number(desired)
+  }
+
+  return current === desired
+}
```

Comment:
Could this be replaced by just `_.isEqual()` ?

Reply:
```js
isEqual(0, '0') // false
```